pkg/server/mcp.go

```diff
@@ -13,18 +13,18 @@ func fromTypeTool[T any](tool mcp.Tool, handler mcp.TypedToolHandlerFunc[T]) (mc
 	return tool, mcp.NewTypedToolHandler(handler)
 }
 
-func NewMCPServer(globals *Globals) *server.MCPServer {
+func NewMCPServer(version string, client *gobuildkite.Client) *server.MCPServer {
```

Comment:
Yeah nice

Reply:
Yeah great when these sorts of things get tightened up. 😅